src/ui_ng/lib/src/repository-stackview/repository-stackview.component.ts

```diff
@@ -153,15 +168,64 @@ export class RepositoryStackviewComponent implements OnInit {
     this.clrLoad(st);
   }
 
+  saveSignatures(event: {[key: string]: string[]}): void {
+    Object.assign(this.signedCon, event);
+  }
+
   deleteRepo(repoName: string) {
+    // get children tags data
+    this.signature = '';
+    if (!(repoName in this.signedCon)) {
```

Comment:
if(this.signedCon[repoName] && this.signedCon[repoName].length >0){
  //Use cache
}else{
  //Fetch
}